src/towncrier/test/test_create.py

```diff
@@ -245,10 +443,12 @@ def test_create_orphan_fragment_custom_prefix(self, runner: CliRunner):
         self.assertEqual(len(fragments), 1)
         change = fragments[0]
         self.assertTrue(change.stem.startswith("$$$"))
-        # Length should be '$$$' characters and 8 random hex characters.
-        self.assertEqual(len(change.stem), 11)
+        # Length should be '$$$' characters, 8 random hex characters, and ".feature".
+        self.assertEqual(len(change.stem), 3 + 8 + len(".feature"))
```

Comment:
Again a change that looks like it's modifying the default behaviour.

Reply:
Now mentioned in release notes